apps/storybook2/src/stories/molecules/AuthWrapper.stories.tsx

```
@@ -0,0 +1,79 @@
+              name="confirmPassword"
+              placeholder="••••••••"
+              type="password"
+            />
+          </div>
+          <div className="flex items-center gap-x-3">
+            <Checkbox id="acceptTerms" name="acceptTerms" />
+            <Label htmlFor="acceptTerms">
+              I accept the&nbsp;
+              <a
+                href="#"
+                className="text-primary-700 hover:underline dark:text-primary-500"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)